packages/hoppscotch-selfhost-web/src/platform/collections/collections.platform.ts

```diff
@@ -70,13 +70,14 @@ function initCollectionsSync() {
 
   gqlCollectionsSyncer.startStoreSync()
 
-  loadUserCollections("REST")
+  // TODO: fix collection schema transformation on backend maybe?
+  // loadUserCollections("REST")
   loadUserCollections("GQL")
 
   // TODO: test & make sure the auth thing is working properly
   currentUser$.subscribe(async (user) => {
     if (user) {
-      loadUserCollections("REST")
+      // loadUserCollections("REST")
```

Comment:
Could you elaborate on why `loadUserCollections` is commented out? Is anything remaining here with the TODO resolution?